style.css

```diff
@@ -104,6 +106,220 @@
   list-style: none;
 }
 
+/* work-section */
+
+.work-section {
+  background-color: #f7f7f9;
+  width: 100%;
+  padding: 30px;
+  padding-top: 100px;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+}
+
+.work-container {
+  display: grid;
+  grid-template-columns: auto;
+  border-radius: 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  align-items: center;
+  padding: 20px;
+  gap: 30px;
+  margin-bottom: 20px;
+}
```

Comment:
I noticed that you are using both, grid for laying out the contents of the individual card. However, as given in the project requirements, please consider using display: flex for styling individual cards and display: gird for the section that contains all the project cards as stated above 🛠🤓 Following is a quote for a related requirement 👇

_In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card._